docs/docs/guides/server-deployment.md

```diff
@@ -172,20 +172,23 @@ By default, `dstack` stores data in plaintext.
 If you want backend credentials and user tokens to be encrypted, set up encryption keys via
 [`~/.dstack/server/config.yml`](../reference/server/config.yml.md#encryption).
 
-## Versioning policy
+## Backward compatibility
```

Comment:
Backward compatibility is only one aspect of the Versioning policy. Versioning/version policy is a standard term:
* https://www.postgresql.org/support/versioning/
* https://kubernetes.io/releases/version-skew-policy/